global_segment_map_node/src/iodb_controller.cpp

```diff
@@ -0,0 +1,591 @@
+// Copyright 2018 Margarita Grinvald, ASL, ETH Zurich, Switzerland
```

Comment:
Why do you put yourself as a sole author?

Reply:
This was copied over from what was in controller.cpp, now I'll just remove these since we have a license file.